src/cephadm/setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+__version__ = '1.0.0'
+
+setuptools.setup(
+     name='cephadm',  
+     version=__version__,
+     scripts=['cephadm'] ,
+     description="cephadm",
+     long_description=(
+	"The cephadm orchestrator is an orchestrator "
+	"module that does not rely on a separate "
+	"system such as Rook or Ansible, but rather "
+	"manages nodes in a cluster by establishing "
+	"an SSH connection and issuing explicit "
+	"management commands."
+	),
+     url="https://github.com/ceph/ceph/tree/master/src/cephadm",
+     license='LGPLv2+',
+     platforms='Linux',
+     packages=setuptools.find_packages(),
+     classifiers=[
+         "Programming Language :: Python :: 3",
```

Comment:
cephadm is deliberately written so it's compatible to python2 and python3. could better off putting
```
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
```
here.